Gemfile.lock

```diff
@@ -200,8 +196,7 @@ GEM
     zeitwerk (2.5.4)
 
 PLATFORMS
-  ruby
-  x86_64-darwin-19
+  x86_64-darwin-20
```

Comment:
I think you have to do the magic to not update platforms - did you do "bundle update" instead of "bundle install" ?

Reply:
@ndushay thanks, have removed this file from the PR